.github/workflows/apply-milestone.yml

```diff
@@ -0,0 +1,68 @@
+name: "[Admin] Milestone Linker"
+
+on:
+  pull_request:
+    types: closed # Trigger on relevant PR events
+    paths: # targeting only code that touches the app
+      - VAMobile/src/**
+
+jobs:
+  process_pr_and_issue:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      issues: write
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Check Linked Issues
+        uses: nearform-actions/github-action-check-linked-issues@v1
+        id: check-linked-issues
+
+      - name: Get those linked issues
+        run: |
+          echo "Linked Issues Count: ${{ steps.check-linked-issues.outputs.linked_issues_count }}"
+          echo "Linked Issues: ${{ steps.check-linked-issues.outputs.issues }}"
+
+      - name: Use linked issues
+        id: found_linked_issues
+        run: |
+          LINKED_ISSUE=${{ steps.check-linked-issues.outputs.issues }}
+          LINKED_ISSUE_NUMBER=$(echo "$LINKED_ISSUE" | cut -d '#' -f 2)
+
+          if [[ -z "$LINKED_ISSUE_NUMBER" ]]; then
+            echo "No explicit linked issue found in PR description for PR #${{ github.event.pull_request.number }}."
+          else
+            echo "Linked issue number found: #$LINKED_ISSUE_NUMBER"
+            echo "linked_issue_number=$LINKED_ISSUE_NUMBER" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Conditional Action Add Milestone to issue or PR
+        if: ${{ always() }}
+        run: |
+          PR_NUM="${{ github.event.pull_request.number }}"
+          LINKED_ISSUE_NUM="${{ steps.found_linked_issues.outputs.linked_issue_number }}"
+          LINKED_ISSUE_N=$(echo "$LINKED_ISSUE_NUM" | sed 's/]$//g')
+          MILESTONE_ID=$(gh api "repos/${{ github.repository }}/milestones?sort=number&direction=asc" --jq '.[0].number')
```

Comment:
@jbergman-oddball are we able to sort these by due_date? This seems to be the order they were created

Reply:
@jbergman-oddball also, we a period of time where the most current milestone isn't the correct one:
- adding issues to Milestone v2.50.0
- v2.50.0 release is cut (no more issues into v2.50.0 allowed)
**- adding issues to Milestone v2.51.0**
- v2.50.0 is released
- Milestone v2.50.0 closes

we might want to instead rework this process, but if there's a way to get the correct milestone that would be great